src/set_theory/cardinal_ordinal.lean

```diff
@@ -13,21 +13,27 @@ import tactic.linarith
 Relationships between cardinals and ordinals, properties of cardinals that are proved
 using ordinals.
 
-## Main definitions and results
+## Main definitions
 
-* The `aleph'` function gives the cardinals listed by their ordinal
-  index, and is the inverse of `aleph_idx`.
+* The function `cardinal.aleph'` gives the cardinals listed by their ordinal
+  index, and is the inverse of `cardinal.aleph_idx`.
   `aleph' n = n`, `aleph' ω = ω`, `aleph' (ω + 1) = ℵ₁`, etc. It is an order isomorphism
   between ordinals and cardinals.
-* The `aleph` function gives the infinite cardinals listed by their
+* The function `cardinal.aleph` gives the infinite cardinals listed by their
   ordinal index. `aleph 0 = ω`, `aleph 1 = succ ω` is the first
   uncountable cardinal, and so on.
```

Comment:
`aleph 1 = succ ω` looks a little misleading in this statement. It means `cardinal.succ cardinal.omega` which is correct, but at first glance it looks like the ordinal `ω + 1`.

Reply:
I agree, especially since we currently don't use `ω` for `cardinal.omega`.